doc/chibi.scrbl

```diff
@@ -1374,6 +1374,8 @@ namespace.
 
 \item{\hyperlink["lib/chibi/repl.html"]{(chibi repl) - A full-featured Read/Eval/Print Loop}}
 
+\item{\hyperlink["lib/chibi/shell.html"]{(chibi shell) - Binding to the most necessary POSIX system calls.}}
```

Comment:
Sorry, I merged before noticing, but this actually doesn't export any system calls or have anything to do with POSIX.

I pushed a new description.

Reply:
My bad, that description wasn't too accurate.

But would you consider hosting documentation on "Github Pages" re-generating on push?
I suspect it would amount to less work than publishing it manually.